hydrus/core/HydrusFileHandling.py

```diff
@@ -490,6 +511,9 @@ def GetMime( path, ok_to_look_for_hydrus_updates = False ):
         
         return HC.TEXT_HTML
         
+    if HydrusText.LooksLikeSVG( bit_to_check ): 
```

Comment:
This comes after the HTML check because an HTML file could contain an SVG but an SVG won't contain the `html` element or doctype tag.